repo/packages/S/spark/0/mantl.json

```diff
@@ -0,0 +1,5 @@
+{
+  "mantl": {
+    "load-balancer": "external"
```

Comment:
Yep, here is an example. Create a json file called spark.json that looks like this:

```json
{
  "name": "spark",
  "config": {
    "mantl": {
      "load-balancer": "external"
    }
  }
}
```

Then, you can deploy it with something like:

```
curl -sku $creds -XPOST -d @spark.json https://mantl-control-01/api/1/install
```